modules/cnv-example-bond-nncp.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_node_network/cnv-configuring-node-network-policy
+
+[id="cnv-example-bond-nncp_{context}"]
+= Example: Bond interface NodeNetworkConfigurationPolicy
+
+[source,yaml]
+----
+apiVersion: nmstate.io/v1alpha1
+kind: NodeNetworkConfigurationPolicy
+metadata:
+  name: bond0-eth1-eth2-policy <1>
+spec:
+  nodeSelector: <2>
+    kubernetes.io/hostname: <node01> <3>
+  desiredState:
+    interfaces:
+    - name: bond0 <4>
+      description: Bond enslaving eth1 and eth2 <5>
+      type: bond <6>
+      state: up <7>
+      ipv4:
+        dhcp: true <8>
+        enabled: true <9>
+      link-aggregation:
+        mode: balance-rr <10>
+        options:
+          miimon: '140' <11>
+        slaves: <12>
+        - eth1 
+        - eth2
+      mtu: 1450 <13>
+----
+<1> Name of the Policy. 
+<2> Optional. If you do not include the `nodeSelector`, the Policy applies to all nodes in the cluster.
+<3> This example uses a hostname node selector.
+<4> Name of the interface.
+<5> Optional. Human-readable description of the interface.
+<6> The type of interface. This example creates a bond. 
+<7> The desired state for the interface after creation.
+<8> Optional. If you do not use `dhcp` you must specify static IP address for the interface.
+<9> Enables `ipv4` in this example.
+<10> The driver mode for the bond. This example uses a round-robin mode.
+<11> Optional. This example uses miimon to inspect the bond link every 140ms.
+<12> The node NICs that will be enslaved to the bond. 
+<13> Optional. The maximum transmission unit (MTU) for the bond. This value is set to `1500` by default, however, many slave links only support an MTU of 1450, where 50 units are reserved for underlying protocols. 
```

Comment:
I think the last part of the explanation (starting from "however, many slave links only support...") is redundant, as the reason for why certain interfaces support a certain value of MTU is not relevant.
I would leave only something like:
Optional. The maximum transmission unit (MTU) for the bond. If not specified, this value is set to `1500` by default.

Reply:
Ah fair enough. 
Updated as suggested.

Reply:
LGTM